.github/workflows/ci.yml

```
@@ -323,7 +323,7 @@ jobs:
       uses: dtolnay/rust-toolchain@stable
 
     - name: Install sccache
-      uses: mozilla-actions/sccache-action@v0.0.4
+      uses: mozilla-actions/sccache-action@v0.0.5
```

Revert a breaking change to get the tests to pass.
```suggestion
      uses: mozilla-actions/sccache-action@v0.0.4
```